<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutSettingDlg</class>
 <widget class="QWidget" name="ShortcutSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="shortcut_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="tip_label">
          <property name="text">
           <string>先按对应的组合键,然后再按Enter确认,ESC退出</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomKeySequenceEdit" name="keySequenceEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomKeySequenceEdit</class>
   <extends>QKeySequenceEdit</extends>
   <header location="global">../../../toolBox/customkeysequenceedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
